src/tests/end-to-end/test-resources/semantics/header-attributes.html

```diff
@@ -0,0 +1,66 @@
+<!--
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License.
+-->
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <title>CodePen - Complex table example</title>
```

Comment:
I think the `<title>` and the `./style.css` link are probably copy-paste leftovers